### PR TITLE
Add load balancer metrics

### PR DESCRIPTION
**Pull request by ErwinSteffens**
Add load balancer metrics. First Golang project so please check it thoroughly ;-)

Not sure about the health status of the targets. I now count them and give the total healthy and unhealthy. 

The API returns the status per target per service. Maybe we can report this more detailed and indicate the status per service.

### CODE REVIEW

**Comment by tboerger** [comment]
Most parts of the pipeline are available again, now I just need to take the time to review this PR :)

**Comment by ErwinSteffens** [comment]
@tboerger Any time to check this out?

File: pkg/exporter/load_balancer.go [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+			prometheus.GaugeValue,
+			float64(loadBalancer.LoadBalancerType.MaxConnections),
+			labels...,
+		)
+
+		c.addMetric(ch, c.ConnectionsPerSecond, "connections_per_second", *loadBalancerMetrics, labels)
+
+		c.addMetric(ch, c.RequestsPerSecond, "requests_per_second", *loadBalancerMetrics, labels)
+
+		c.addMetric(ch, c.IncomingBandwidth, "bandwidth.in", *loadBalancerMetrics, labels)
```
Comment (tboerger) [inline pkg/exporter/load_balancer.go:306]: I think the dot is wrong here, right?

File: pkg/exporter/load_balancer.go [hunk 2]
```diff
@@ -0,0 +1,476 @@
...
+
+		c.addMetric(ch, c.OutgoingBandwidth, "bandwidth.out", *loadBalancerMetrics, labels)
```
Comment (tboerger) [inline pkg/exporter/load_balancer.go:308]: Same here

File: pkg/exporter/load_balancer.go (hunk 1 above)
Comment (ErwinSteffens) [reply pkg/exporter/load_balancer.go]: Yes, fixed

File: pkg/exporter/load_balancer.go (hunk 2 above)
Comment (ErwinSteffens) [reply pkg/exporter/load_balancer.go]: Yes, fixed

**Comment by tboerger** [comment]
@ErwinSteffens sadly the linter doesn't like some change. After fixing that I tend to merge the PR and play around with the new metrics. Maybe I will apply some change afterwards, but at least you got something to work with as the latest tag schould be directly built.

**Comment by ErwinSteffens** [comment]
I think I have fixed all lint issues. Does the action automatically trigger again?

**Pull request merged**